pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -985,18 +985,33 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, allowEmul
 		}
 	}
 
+	cacheClient := cache.CacheCreator{}
+	vmConfigurator := netsetup.NewVMNetworkConfigurator(vmi, cacheClient)
 	ifaceNamingScheme := namescheme.CreateNetworkNameScheme(vmi.Spec.Networks)
 	for _, network := range netvmispec.NetworksToHotplugWhosePodIfacesAreReady(vmi) {
 		log.Log.Infof("will hot plug %s", network.Name)
+		if err := vmConfigurator.StartDHCP(domain, network); err != nil {
```

Comment:
Please re-use `SetupPodNetworkPhase2` instead of re-implementing it. Same as was done with `SetupPodNetworkPhase1`